src/desktop/apps/purchase/server.tsx

```diff
@@ -0,0 +1,59 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { stitch } from "@artsy/stitch"
+import { routes } from "reaction/Apps/Purchase/routes"
```

Comment:
Didn't notice this before, but the naming (singular `purchase`) reads weird to me, since it is either referring to a single purchase or the team.

Can we rename the folders/naming throughout this PR to `purchases`?

Reply:
I know this is referring to a reaction import that won't be easy to update this go-around, but everything within force should be fine. 😄

Reply:
Best way to handle this would be to make a major release of reaction with the app renamed. Then in the auto PR to force, update the reaction paths. 

But yeah, not related to this PR 😄

Reply:
So just to be clear should I or should I not rename everything in this PR?

Reply:
Please rename the files within force, but no need to update the name of the reaction app at this moment.

Reply:
Iirc I think we went with the singular to be consistent with the other apps.

Reply:
The app names should reflect the entity (or entities) that are being served! So we have both `artists` and `artist`. I would expect an `order` app to be the one where you create an order, but an `orders` app where you see a list of them.

In this case we've chosen to name it `purchases`, which is OK but I do think the plural makes more sense given it's serving multiple.

Reply:
I thought we have already decided [what the app should be called](https://artsy.slack.com/archives/C9YNS4X32/p1579102785143700?thread_ts=1579101779.142500&cid=C9YNS4X32)? 😄